### ISSUE TITLE
Custom fonts

### ISSUE THREAD

**Issue by fulldecent** [codestergit/SweetAlert-iOS#35]
Please add custom fonts

**Comment by fulldecent** [comment]
@codestergit To be clear, are you still accepting contributions to this project?? If so would a PR that fixes this be in scope to be merged?

**Comment by codestergit** [comment]
@fulldecent Yes I accept the PR. But custom font is out of scope. You can customize this alert at your own end. If you want to send PR please send. We can discuss if it adds something to this project.

Thanks for your time
